src/components/CallView/shared/VideoBottomBar.vue

```diff
@@ -171,7 +172,19 @@ export default {
 		},
 
 		stopFollowingLabel() {
-			return t('spreed', 'Stop following')
+			return t('spreed', `Stop following`)
```

Comment:
It should use single quotes instead of backticks.

Reply:
So the rebase failed @ma12-co xD